HARK/interpolation.py

```diff
@@ -4211,6 +4211,45 @@ def _derY(self, x, y):
         return dfdy
 
 
+class DiscreteInterp(MetricObject):
+    """
+    An interpolator for variables that can only take a discrete set of values.
+
+    If the function we wish to interpolate, f(args) can take on the list of
+    values DiscreteVals, this class expects an interpolator for the index of
+    f's value in DiscreteVals.
+    E.g., if f(a,b,c) = DiscreteVals[5], then IndexInterp(a,b,c) = 5.
+
+    Parameters
+    ----------
+    IndexInterp: HARKInterpolator
```

Comment:
are these parameters invoking a mathematical notion in their variable names? if not please use `snake_case` for variable/argument names